spec/consul_support/manifests/test.pp

```diff
@@ -0,0 +1,33 @@
+class libkv::test(
+$url = 'mock://'
+) {
+  $supports = libkv::supports({'url' => $url})
+  notify { "supports = ${supports}": }
+  $provider = libkv::provider({'url' => $url})
+  notify { "provider = ${provider}": }
+  $loopvar = {
+    '/meats/pork'    => 'test1',
+    '/meats/chicken' => 'test4',
+    '/meats/turkey'  => 'test5',
+    '/meats/beef'    => 'test2',
+    '/fruits/apple'  => 'test3',
+    '/fruits/banana' => 'test4',
+  }.each |$key, $value| {
+          libkv::put({ 'url' => $url, 'key' => $key, 'value' => $value});
+    $get = libkv::get({'url' => $url, 'key'       => $key});
+          notify { "${key} get = ${get}": }
+    $atomic_get = libkv::atomic_get({'url' => $url, 'key' => $key});
+          notify { "${key} atomic_get = ${atomic_get}": }
+    libkv::atomic_put({'url'            => $url, 'key' => $key, 'value' => 'testzor', 'previous' => $atomic_get});
+    $atomic_put = libkv::atomic_get({'url' =>  $url, 'key'       => $key});
```

Comment:
Have some arrow alignment issues in this file.

Reply:
This is a legacy file I haven't done anything with.  Can we fix it when we implement the consul_plugin?